org.eclipse.xtext.full.releng/pom.xml

```diff
@@ -33,6 +33,7 @@
 		<module>../org.eclipse.xtext.ide</module>
 		<module>../org.eclipse.xtext.testing</module>
 		<module>../org.eclipse.xtext.xtext.ide</module>
+		<module>../org.eclipse.xtext.generator.dependencies</module>
```

Comment:
Should this be `xtext.xtext.generator.dependencies` ?

Reply:
`org.eclipse.xtext.xtext.generator.additional.bundles` would be more correct but even longer ...

Reply:
Yes, please

Reply:
`org.eclipse.xtext.xtext.generator.dependencies` that is

Reply:
@szarnekow rename done.